AlphaWallet/Transfer/Coordinators/TransactionConfirmationCoordinator.swift

```diff
@@ -111,6 +112,23 @@ class TransactionConfirmationCoordinator: Coordinator {
 }
 
 extension TransactionConfirmationCoordinator: TransactionConfirmationViewControllerDelegate {
+
+    func viewDidAppear(_ controller: TransactionConfirmationViewController) {
+        switch viewModel {
+        case .dappTransaction, .tokenScriptTransaction:
+            break
+        case .sendFungiblesTransaction(let sendFungiblesViewModel):
+            switch sendFungiblesViewModel.transferType {
+            case .nativeCryptocurrency, .ERC20Token:
+                analyticsCoordinator?.log(action: Analytics.Action.actionSheetShowsUp)
```

Comment:
I think we can log this in `start()` and without the checks for the type. The issue/comment originally talked about "actionsheet for normal transactions" only, but since we have done it for most types of transactions, lets just log all of them.